File: src/codes/Users_Manual/modulation/custom.cpp
//~#include<autd3.hpp>
#include <autd3/modulation/custom.hpp>

//~#include<vector>
//~int main() {
std::vector<uint8_t> buf = {0xFF, 0x00};
const auto m = autd3::modulation::Custom(std::move(buf), 4000 * autd3::Hz);
//~return 0; }

File: src/codes/Users_Manual/modulation/custom.rs
# use autd3::prelude::*;
# use autd3::modulation::Custom;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let buf = vec![0xFF, 0x00];
let m = Custom::new(buf, 4000 * Hz);
# Ok(())
# }